deployer/templates/hawkular-cassandra-node-dynamic-pv.yaml

```diff
@@ -114,6 +114,12 @@ objects:
             exec:
               command:
               - "/opt/apache-cassandra/bin/cassandra-docker-ready.sh"
+          lifecycle:
+            preStop:
+              exec:
+                command:
+                - "/opt/apache-cassandra/bin/cassandra-prestop.sh"
+          terminationGracePeriodSeconds: 1800
```

Comment:
You can't get the terminationGracePeriodSeconds from the downward api. The downward only provides a limited number of parameters: http://kubernetes.io/docs/user-guide/downward-api/